packages/cursorless-vscode/src/keyboard/KeyboardCommandsTargeted.ts

```diff
@@ -261,6 +261,18 @@ export default class KeyboardCommandsTargeted {
       // For some Cursorless actions, it is more convenient if we automatically
       // exit modal mode
       await this.modal.modeOff();
+    } else {
+      // If we're not exiting cursorless mode, preserve the keyboard mark
+      // FIXME: Better to just not clobber the keyboard mark on each action?
```

Comment:
i think it is funny that we outlawed TODO so now just use FIXME.

Reply:
TODO is for things that must be fixed before merge; FIXME is for things we don't like but are ok to merge. I guess we should prob add a note about that to contributor docs

Fwiw we've always tried to use this convention, but now it's enforced so those TODOs can actually give peace of mind